src/deepsparse/transformers/helpers.py

```diff
@@ -76,15 +76,17 @@ def get_deployment_path(model_path: str) -> Tuple[str, str]:
         deployment_path = zoo_model.deployment_directory_path
         return deployment_path, os.path.join(deployment_path, _MODEL_DIR_ONNX_NAME)
     elif model_path.startswith("hf:"):
+        # load Hugging Face model from stub
```

Comment:
now that we just grab the entire deployment directory, wonder if we can just use model_to_path here now, or even need the function at all.  either way, would be good to move this to a helper function since it's the same as the changes below

Reply:
i'm good with just calling model_to_path here. i'm kicking around the idea of making this snapshot_download more intelligent so it will only download required files and fail before downloading if required files (model.onnx) aren't present